### ISSUE TITLE
Upload lyrics to lrclib

### ISSUE THREAD

**Issue by bt4ibwem8** [shub39/Rush#75]
What about option to upload lyrics? So many users could easily contribute to the database...

**Comment by shub39** [comment]
Good idea! A bit busy atm, this will take time and a lot of figuring out.

**Comment by shub39** [comment]
it is and i doubt it'll be used much. Its quite hard already to make space for adding new features
